cmd/generate-plugin-overview/generate-plugin-overview.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2019 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// plugin-overview reads the manifests in a directory and creates a markdown overview page
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/golang/glog"
+
+	"sigs.k8s.io/krew/pkg/index"
+	"sigs.k8s.io/krew/pkg/index/indexscanner"
+)
+
+const (
+	separator  = " | "
+	pageHeader = `# Plugin overview
```

Comment:
Let's do something bit more useful as we'll likely link to this
document from Krew's main repo readme.

```
## Available kubectl plugins

The following kubectl plugins are currently available
on [Krew plugin index](https://sigs.k8s.io/krew-index).

To install these kubectl plugins:

1. [Install Krew](link)
2. Run `kubectl krew install PLUGIN_NAME` to install them via Krew.


```

Reply:
Yes, that makes sense.